db/migrate/20230307102857_add_points_to_employee_table.rb

```diff
@@ -1,5 +1,5 @@
 class AddPointsToEmployeeTable < ActiveRecord::Migration[6.1]
   def change
-    add_column :employees, :number_of_earned_points, :integer, default: 0
+    add_column :employees, :number_of_earned_points, :float, default: 0
```

Comment:
Why do you want to store number of points using float?

Reply:
But the price is a decimal. So you may end up in situation that something costs 3,5$ so this way the points can be correctly displayed.

Reply:
Good point, but why not decimal then?

Reply:
Too complicated I guess ;) I haven't really thought about it. Float was the first thing that came to my mind.